docs/source/aws-backend.rst

```diff
@@ -0,0 +1,90 @@
+It is recommended to restrict the address range to a subnet including the public IP of the machine executing tests with the AWS backend.
+For example ``<external-ip>/24``.
+
+Accessing cluster nodes
+-----------------------
+
+SSH can be used to access cluster nodes for the purpose of debugging if :paramref:`~dcos_e2e.backends.AWS.workspace_dir` is set.
+The AWS backend generates a key in the :paramref:`~dcos_e2e.backends.AWS.workspace_dir` directory under ``ssh/id_rsa``.
+Adding this key to the ``ssh-agent`` or changing its file permissions to ``400`` will allow for connecting to the cluster via the ``ssh`` command.
+The SSH user depends on the :paramref:`~dcos_e2e.backends.AWS.linux_distribution` given to the :py:class:`~dcos_e2e.backends.AWS` backend.
+For :py:obj:`~dcos_e2e.distributions.Distribution.CENTOS_7` that is ``centos``.
+
+It is important to keep in mind files in the given :paramref:`~dcos_e2e.backends.AWS.workspace_dir` are temporary and are removed up when the cluster is destroyed.
+If :paramref:`~dcos_e2e.backends.AWS.workspace_dir` is unset the :py:class:`~dcos_e2e.backends.AWS` backend will create a new temporary directory in an operating system specific location.
+
+Cluster lifetime
+----------------
+
+The cluster lifetime is fixed at a maximum of two hours.
+That is because of a limitation of `dcos-launch`_ which is used under the hood.
+That will most likely change in the future.
+
+If the cluster was launched with ``maws`` (Mesosphere temporary AWS credentials) the cluster can be controlled via `CCM`_.
+This allows for extending the cluster lifetime and also for cleaning up the cluster if anything goes wrong.
+
+EC2 instance types
+------------------
+
+The AWS backend does not offer a choice for EC2 instances.
+Currently it launches ``m4.large`` instances for DC/OS nodes.
+
+Unsupported features
+--------------------
+
+The AWS backend does currently not support the :py:class:`~dcos_e2e.cluster.Cluster` feature of copying files to the DC/OS installer.
+The progress on this feature is tracked in :issue:`DCOS-21894`.
+
+Troubleshooting
+---------------
+
+In case of an error during the DC/OS installation the journal from each node will be dumped and downloaded to the folder that the tests were executed in.
+The logs are prefixed with the installation phase that failed, ``preflight``, ``deploy`` or ``postflight``.
+
+When using temporary credentials it is required to pay attention that the credentials are still valid or renewed when destroying a cluster.
+If the credentials are not valid anymore the AWS backend does not delete the public/private key pair generated during cluster creation.
+It is therefore recommended to a periodically renew temporary AWS credentials when executing tests using the AWS backend.
+
+In rare cases it might also happen that a AWS stack deployment fails with the message ``ROLLBACK_IN_PROGRESS``.
+In that case one of the EC2 instances failed to come up. Re-running the test is the only option then.
```

Comment:
New sentence new line.

"Re-running the test" - let's not talk about tests here - this is a library for creating and managing an AWS DC/OS cluster. It might be used in non-test environments.